configurations/wikidata-animals.ttl

```diff
@@ -15,7 +15,14 @@
 #####################################
 
 <https://linked.opendata.cz/resource/knowledge-graph-browser/configuration/wikidata/animals> a browser:Configuration ;
-  dct:title "Configuration for browsing animals from Wikidata"@en ;
+  dct:title "Browsing animals and plants from Wikidata"@en,
+   "Procházení živočichů a rostlin z Wikipedie"@cs;
+  dct:description "Classification of all organisms, plants and animals, into taxa."@en,
+   "Možnost procházet sítí taxonů všech zvířat a rostil jež jsou na Wikipedii."@cs;
+  browser:hasVisualStyleSheet <https://linked.opendata.cz/resource/knowledge-graph-browser/wikidata/animals/style-sheet>;
+  browser:startingNode <http://www.wikidata.org/entity/Q7377>,
+   <http://www.wikidata.org/entity/Q192154>;
+  browser:resourceUriPattern "^http://www\.wikidata\.org/entity/Q[1-9][0-9]*$";
```

Comment:
This means that when the user's search query matches this pattern, the part of the pattern matched by the query is replaced by the query?

Reply:
I'm not sure what you mean exactly. This should help the user 1) decide whether the IRI he/she typed is correct before sending it to the server and 2) for constructing the whole IRI from the search string (the idea is to somehow replace the non-terminals with a search string and check if the IRI is valid).

Reply:
*and it should be IRI, not URI